cypress/integration/helipopper.spec.ts

```diff
@@ -29,7 +29,7 @@ describe("@ngneat/helipopper", () => {
   describe("Custom template", () => {
     it("should create tooltip with a custom template", () => {
       cy.get("#custom-template button")
-        .click()
+        .click({ force: true })
```

Comment:
Why did you change it?

Reply:
I changed it because the button is out of view and forcing the click skips the actionability check, which imho is not essential here.